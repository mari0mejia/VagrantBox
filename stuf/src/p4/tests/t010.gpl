string window_title = "p4 t010.gpl -- compare to t010.jpg ***PRESS q TO QUIT***";
rectangle my_rectangle(x = 100, y = 100, rotation=-30, w=45);
circle    my_circle(x = 200, y = 200, radius=30);
triangle    my_triangle(x = 300, y = 100, size=40, rotation=30);
pixmap    my_pixmap(x = 100, y = 400, green=0.5);
textbox    my_textbox(x = 200, y = 300, text = "Home Sweet Home");
