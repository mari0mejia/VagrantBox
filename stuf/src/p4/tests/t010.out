gpl.cpp::main()
  input file(tests/t010.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
circle my_circle
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(60),
    proximity = int(4),
    radius = int(30),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(60),
    x = int(200),
    y = int(200)
}
pixmap my_pixmap
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(10),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(100),
    y = int(400)
}
rectangle my_rectangle
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(-30),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(45),
    x = int(100),
    y = int(100)
}
textbox my_textbox
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string("Home Sweet Home"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(200),
    y = int(300)
}
triangle my_triangle
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(44),
    proximity = int(4),
    red = double(0),
    rotation = double(30),
    size = int(40),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(40),
    x = int(300),
    y = int(100)
}
string window_title = "p4 t010.gpl -- compare to t010.jpg ***PRESS q TO QUIT***"
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
circle my_circle
{
    animation_block = NULL,
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(60),
    proximity = int(4),
    radius = int(30),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(60),
    x = int(200),
    y = int(200)
}
pixmap my_pixmap
{
    animation_block = NULL,
    blue = double(0.5),
    drawing_order = int(0),
    filename = string(""),
    green = double(0.5),
    h = int(50),
    proximity = int(4),
    red = double(0.5),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(100),
    x = int(100),
    y = int(400)
}
rectangle my_rectangle
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(10),
    proximity = int(4),
    red = double(1),
    rotation = double(-30),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(45),
    x = int(100),
    y = int(100)
}
textbox my_textbox
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    spacing = int(10),
    text = string("Home Sweet Home"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(140),
    x = int(200),
    y = int(300)
}
triangle my_triangle
{
    animation_block = NULL,
    blue = double(0),
    drawing_order = int(0),
    green = double(1),
    h = int(44),
    proximity = int(4),
    red = double(0),
    rotation = double(30),
    size = int(40),
    skew = double(1),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(40),
    x = int(300),
    y = int(100)
}
string window_title = "p4 t010.gpl -- compare to t010.jpg ***PRESS q TO QUIT***"
